x-pack/test/cases_api_integration/common/lib/api/attachments.ts

```diff
@@ -128,7 +128,7 @@ export const createCaseAndBulkCreateAttachments = async ({
 
 export const getAttachments = (numberOfAttachments: number): BulkCreateCommentRequest => {
   return [...Array(numberOfAttachments)].map((_, index) => {
-    if (index % 0) {
+    if (index % 10 === 0) {
```

Comment:
`index % 0` will result in `NaN` because it is dividing by 0. So I don't believe this if block would get executed. I think we wanted to create a user comment when the remainder is 0 so I'm using `10` here. Happy to change it to something else though.

Reply:
Why every 10th? 😄 

And if this was never run before, do we even need it?

Reply:
haha that was my best guess on what we were trying to do with `index % 0`. @cnasikas what do you think, should we remove the user comment section?